src/main/java/tutorspet/logic/commands/StatisticsCommand.java

```diff
@@ -0,0 +1,85 @@
+package tutorspet.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static tutorspet.commons.core.Messages.MESSAGE_INVALID_MODULE_CLASS_DISPLAYED_INDEX;
+import static tutorspet.commons.core.Messages.MESSAGE_INVALID_STUDENT_DISPLAYED_INDEX;
+import static tutorspet.commons.util.CollectionUtil.requireAllNonNull;
+import static tutorspet.logic.parser.CliSyntax.PREFIX_CLASS_INDEX;
+import static tutorspet.logic.parser.CliSyntax.PREFIX_STUDENT_INDEX;
+import static tutorspet.logic.util.ModuleClassUtil.getAbsentWeek;
+import static tutorspet.logic.util.ModuleClassUtil.getParticipationScore;
+
+import java.util.List;
+
+import tutorspet.commons.core.index.Index;
+import tutorspet.logic.commands.exceptions.CommandException;
+import tutorspet.model.Model;
+import tutorspet.model.moduleclass.ModuleClass;
+import tutorspet.model.student.Student;
+
+/**
+ * Gives a summary of a student's attendance for a specific class in the student manager.
+ */
+public class StatisticsCommand extends Command {
+
+    public static final String COMMAND_WORD = "stats";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "Gives a summary of a student's attendance"
+            + "for a specific class\n"
+            + "Note: All indexes and numbers must be a positive integer.\n"
+            + "Parameters: "
+            + PREFIX_CLASS_INDEX + "CLASS_INDEX "
+            + PREFIX_STUDENT_INDEX + "STUDENT_INDEX ";
+
+    public static final String MESSAGE_SUCCESS = "%1$s attendance summary for %2$s:\n"
+            + "Average participation: %3$s\n"
+            + "Lesson(s) not attended:\n"
+            + "%4$s";
```

Comment:
Would it be better to indicate 'weeks'? So index 4 will be name of the lesson and index 5 will be the string of weeks not attended.
```suggestion
            + "%4$s, Weeks: %5$s";
```

Reply:
not sure if thursday 2pm - 4pm: weeks 1 2 3 4 5 would look better?

Reply:
Can!